app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -29,6 +29,15 @@
                 android:text="@string/create_account"/>
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/createAccountWarning"
```

Comment:
Please use 8 spaces for indentation to be consistent with the rest of the file

Reply:
Done... that was an auto created entry by AndroidStudio